### ISSUE TITLE
a11y: Fix heading structure to begin with h3?

### ISSUE THREAD

**Issue by ambersdai** [openstax/cnx#115]
## Overview
From https://drive.google.com/file/d/0BxiTbl6K_ZNRelpxRlkwYkZnb3c/view

### Issue 3: The heading structure is broken
The sections have headings that identify them (1.3.1) but headings are marked such that they cannot be
programmatically identified.
Example with empty headings: the structure of headings is broken and there are two empty headings

### Reference to standards
1.3.1 Info and Relationships: Information, structure, and relationships conveyed through presentation can be
programmatically determined or are available in text. (Level A)
### Suggestion
- Make headings to organize an HTML page and avoid empty headings. Use an h1 element for the overall title, and h2 elements for major sections, and h3 elements for sub-sections. 
- Alternatively, you can use multiple h1 headings, but each of them has to be a heading to a separate section or article. Avoid skipping the levels of the headings, e.g. if you have h1, it can only be followed by h2, which can only be followed by h3 etc. Use Web Accessibility Toolbar for IE to find defects on your website.

![](https://trello-attachments.s3.amazonaws.com/581c83e5cae45416fa9bf5e2/5a7df65e0db2cb4f3b1e2c30/55567e29d0b7be8cbb0ab6be755d14d7/image.png)

![](https://trello-attachments.s3.amazonaws.com/581c83e5cae45416fa9bf5e2/5a7df65e0db2cb4f3b1e2c30/b0f2dd4fabf11ff32b308e2109b2b848/image.png)

## PRs
What PRs need to be merged to complete this task/feature
## Test Plan
How will QA know this feature works? What parts of regression tests would be most affected?
## Acceptance Criteria
Expected behavior

**Comment by ambersdai** [comment]
Seems like this stems from the CNXML->XHTML transform. The HTML page does not know that it will be in webview so it uses a higher heading level.

Also, as part of baking we update the heading levels so it might need to be fixed in there as well - Philip

**Comment by ambersdai** [comment]
Decision: Alina will have Helene take a look in Q4, may slide to CNX.
